DXRModules/DeusEx/Classes/DXRBingoCampaign.uc

```diff
@@ -0,0 +1,258 @@
+class DXRBingoCampaign extends DXRActorsBase transient;
+
+function PreFirstEntry()
+{
+    local DXREvents events;
+    local BlackHelicopter jock;
+    local Switch1 button;
+    local PayPhone invisibleWall;
+    local #var(prefix)FlagTrigger ft;
+
+    if (!dxr.flags.IsBingoCampaignMode()) return;
+    foreach AllActors(class'DXREvents', events) break;
+    if (events == None) return;
+
+    Super.PreFirstEntry();
+
+    switch (dxr.localURL) {
+        case "02_NYC_BATTERYPARK":
+            dxr.flags.settings.starting_map = 20;
+            events.CreateBingoBoard();
```

Comment:
bingoBoardRoll should probably be reset back to 0 before each CreateBingoBoard, so that rerolls on individual missions don't effect the starting bingo board on each subsequent mission (ie. the rerolls will only effect the single mission they were rerolled on).  This would make things more consistent if people ever decided to do a race of this.